MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+# Maintainers
+
+This file lists the maintainers for the Habitat core plans.
+
+The maintainers for the Habitat project are listed in the
+[Habitat Maintainers list](https://github.com/habitat-sh/habitat/blob/master/MAINTAINERS.md).
+
+To become a maintainer, open a pull request to this list.
+
+## Lieutenant
+
+* [Adam Jacob](https://github.com/adamhjk)
+
+## Maintainers
+
+* [Dave Parfitt](https://github.com/metadave)
+* [Elliott Davis](https://github.com/elliott-davis)
+* [Fletcher Nichol](https://github.com/fnichol)
+* [Jamie Winsor](https://github.com/reset)
```

Comment:
Yes we should. I copy/pasted this from the history of the other repo. Will update.